### DISCUSSION TITLE
data-feathers don't show up in child rows of Datatables.

### DISCUSSION THREAD

**Discussion by peterimbery** [cdnjs/cdnjs#14087] [General] [answered]
I have a datable that I display some data feathers in. I do a feather.replace() in the drawCallback function. The table is a responsive table. If the column  I am trying to display is in the responsive child row the data feather is not shown. If it is visible on the table the data feather appears just fine. If I make the browser bigger and smaller so that column moves from visible to to the child row it behaves the same way. 

When the column is visible the <i data-feather="help-circle"></i> renders to a svg just fine. If it ends up in the child row the computed output just stays at <i data-feather="help-circle"></i> and no svg is computed. 

Is there an option or do I have to do a feather.replace somewhere else as well?

**Comment by MattIPv4** [comment, marked as answer]
Hi there,

We are a host for many libraries -- we do not develop these libraries ourselves, and as such, do not provide support for the libraries.

You will need to reach out to the specific library you are using for support.
